draft-ietf-tls-tls13.md

```diff
@@ -273,10 +273,6 @@ that has two basic properties:
 - The connection is reliable.  Messages include an authentication
   tag which protects them against modification.
 
-Note: The TLS Record Protocol can operate in an insecure mode but is generally
-only used in this mode while another protocol is using the TLS Record
-Protocol as a transport for negotiating security parameters.
-
```

Comment:
You don't need to say anything at all.  It's the introduction.  How much of that is more important or even as important as things like "servers are always authenticated and clients are sometimes authenticated" or "messages have confidentiality and integrity protection" or "0-RTT is fast, but risky".